installer/installer.go

```diff
@@ -263,26 +264,42 @@ func (s *Stack) allocateDomain() error {
 	return nil
 }
 
+func (s *Stack) loadKeyPair(name string) error {
+	keypair, err := loadSSHKey(name)
+	if err != nil {
+		return err
+	}
+	fingerprint, err := awsutil.FingerprintImportedKey(keypair.PrivateKey)
+	if err != nil {
+		return err
+	}
+	res, err := s.ec2.DescribeKeyPairs(&ec2.DescribeKeyPairsRequest{
+		Filters: []ec2.Filter{
+			{
+				Name:   aws.String("fingerprint"),
+				Values: []string{fingerprint},
+			},
+		},
+	})
+	if err != nil {
+		return err
+	}
+	if len(res.KeyPairs) == 0 {
+		return errors.New("No matching key found")
+	}
+	s.SSHKey = keypair
```

Comment:
This should use the name from the response as there could be a matching key of another name.